__tests__/mongo.test.js

```diff
@@ -0,0 +1,63 @@
+const { filesystem } = require('gluegun')
+const { expect } = require('chai')
+const cliCommand = require('../helpers/cli-command')
+const checkIfStringExistsInFile = require('../helpers/check-if-string-exists-in-file')
+const generateRandomProjectName = require('../helpers/generate-random-project-name')
+
+const projectName = generateRandomProjectName()
+
+describe('Mongo layer', () => {
+  afterEach(() => {
+    filesystem.remove(projectName)
+  })
+
+  it('should create a project with mongodb dependency', async () => {
+    // Given
+    await cliCommand(`new --name ${projectName} --mongo`)
+
+    // When
+    const packageFile = filesystem.read(`${projectName}/package.json`)
+    const mongodbDependencyExists = checkIfStringExistsInFile(packageFile, '"mongodb":')
```

Comment:
I think we could to load it as a JSON as this[file](https://github.com/herbsjs/herbs-cli/blob/cde7dda635b072937daacec253194e37f6ad7c92/__tests__/packagejson.test.js#L18)